client/lib/media/test/validation-store.js

```diff
@@ -340,5 +353,30 @@ describe( 'MediaValidationStore', function() {
 				101: [ MediaValidationErrors.FILE_TYPE_UNSUPPORTED ]
 			} );
 		} );
+
+		it( 'should detect an external media error and set error on site 0', () => {
```

Comment:
Yep that's right, and good point. I've renamed the constant and the test to be clearer. I don't want to export the constant as it has no use outside the and so I've redefined it in the test - should it ever change then the test will fail.